debian/mariadb-common.postinst

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+case "$1" in
+  configure)
+    # New packaging paradigm for my.cnf handling among MySQL variants
+    # Used in Ubuntu since Dec-2014 and in Debian since Jul-2015
+    #
+    # If the new mysql-common package does not provide
+    # the update-alternatives facility, notify user about manual fall back
+    if [ -f /usr/share/mysql-common/configure-symlinks ]
+    then
+      /usr/share/mysql-common/configure-symlinks install mariadb "/etc/mysql/mariadb.cnf"
+    else
+      # As configure can be called many times, don't re-create the symlink
+      # if it is there already
+      if [ ! -L /etc/mysql/my.cnf ]
+      then
+        echo "Notice: configure-symlinks trigger could not be called."
+        echo "Please manually create symlinks by running: "
+        echo "  mv -f /etc/mysql/my.cnf /etc/mysql/my.cnf.old"
+        echo "  ln -sf mariadb.cnf /etc/mysql/my.cnf"
```

Comment:
Why cannot you do it automatically here?

Reply:
We are already automatically installing the symlink. So as such, there is "nothing to automate". This text will only ever be displayed if the normal installation failed horribly. Keeping echoing here is thus sensible, but very marginal. The main reason I include this is to keep the packaging as identical with downstream as possible. This section might be removed in the future from both upstream and downstream completely, since the `if [ ! -L /etc/mysql/my.cnf ]` should never evaluate true and this section would never be shown.